views/index.ejs

```diff
@@ -4,6 +4,7 @@
     <meta charset='utf-8'>
     <title><%= _('Qwant Maps') %>
       <% if(config.app.versionFlag) { %> - <%= config.app.versionFlag %><% } %>
+      <% if(config.envName) { %> - <%= config.envName %><% } %>
```

Comment:
I would recommend to add `envName` to default_config.yml in order to make all allowed keys explicit.

Reply:
Done, thanks for the suggestion.